### ISSUE TITLE
Hackage unable to build package.

### ISSUE THREAD

**Issue by jonathanknowles** [input-output-hk/cardano-coin-selection#71]
Hackage is unable to build version [`1.0.0`](https://hackage.haskell.org/package/cardano-coin-selection-1.0.0) of the [`cardano-coin-selection` package](https://hackage.haskell.org/package/cardano-coin-selection).

Extract from build log:
```hs
src/library/Cardano/CoinSelection/Fee.hs:351:31: error:
    • Couldn't match type ‘m’ with ‘Data.Functor.Identity.Identity’
      ‘m’ is a rigid type variable bound by
        the type signature for:
          senderPaysFee :: forall i o (m :: * -> *).
                           (Ord i, MonadRandom m) =>
                           FeeOptions i o
                           -> CoinMap i
                           -> CoinSelection i o
                           -> ExceptT (FeeAdjustmentError i o) m (CoinSelection i o)
        at src/library/Cardano/CoinSelection/Fee.hs:(334,1)-(339,61)
      Expected type: StateT
                       (CoinMap i)
                       (ExceptT (FeeAdjustmentError i o) m)
                       (CoinSelection i o, Fee)
        Actual type: StateT
                       (CoinMap i)
                       (ExceptT (FeeAdjustmentError i o) Data.Functor.Identity.Identity)
                       (CoinSelection i o, Fee)
```
Source: https://hackage.haskell.org/package/cardano-coin-selection-1.0.0/reports/1

**Comment by jonathanknowles** [comment]
I've made a [metadata revision](https://hackage.haskell.org/package/cardano-coin-selection-1.0.0/revisions/) on Hackage, specifying a lower version bound of `0.5.6.0` for the `transformers` library.

I'll leave this issue open until I can confirm that Hackage is able to build the library.

**Comment by jonathanknowles** [comment]
I've added a request to rebuild the package [here](https://github.com/haskell-CI/hackage-matrix-builder/issues/32).

**Comment by jonathanknowles** [comment]
This was fixed with the [v1.0.1 release](https://hackage.haskell.org/package/cardano-coin-selection-1.0.1).
